crates/circuit/src/operations.rs

```diff
@@ -311,13 +309,13 @@ static STANDARD_GATE_NAME: [&str; STANDARD_GATE_SIZE] = [
     "cu",           // 39
     "cu1",          // 40
     "cu3",          // 41
-    "c3x",          // 42
+    "mcx",          // 42 ("mcx")
     "c3sx",         // 43
-    "c4x",          // 44
+    "c4x",          // 44 ("mcx")
```

Comment:
Doesn't this need to be `mcx` too?

Reply:
Ok, changed in [8c922aa](https://github.com/Qiskit/qiskit/pull/12659/commits/8c922aa5031b6bdf9c0782869156eba8ffe210f9).